csuntechlab/iSTART
# STR-227 -- Navigation menu on mobile devices are not able to click the full container

# Description
Fixed navigation bug on mobile where the router-link was only able to click on the text and not the whole container.
  
# Testing
1. Open the navigation menu on mobile, ensure wherever you hover, you are able to click the proper route.
2. Ensure styling is not broken on mobile and desktop.
3. Check if all functionality for navigation works as intended on all viewports.
